### ISSUE TITLE
peer dependency conflicts with clsx

### ISSUE THREAD

**Issue by dez1729** [goetzrobin/spartan#207]
### Please provide the environment you discovered this bug in.

ui/core 0.0.1-alpha.336

### Which area/package is the issue in?

Don't know / other

### Description

It seems that the installed version of `clsx` which is 1.2.1 conflicts with the current version which is 2.1.0. As a result, any npm updates fail unless using `--force`. 

![image](https://github.com/goetzrobin/spartan/assets/20635596/817daef1-c49d-494e-82fb-039c696b1ed4)

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by goetzrobin** [comment]
Hey @dez1729! Have you had clsx installed before or was it installed as part of the spartan CLI setup?

**Comment by dez1729** [comment]
i installed clsx in my project separately which is the reason the conflict occurred. I would need it installed to use clsx on the installed spartan packages correct?

**Comment by goetzrobin** [comment]
yes. the CLI installs the correct version for you, but if your project already uses clsx then it might cause conflicts. Let me see what breaking changes were introduced and if it's a good idea for bumping up the version in general
